### ISSUE TITLE
Conflict with anaconda installation on OSX

### ISSUE THREAD

**Issue by nickeubank** [conda-forge/fiona-feedstock#46]
I think that something in the fiona feedstock is causing conflicts with the default anaconda python installation in osx. It's mostly been coming up in the [geopandas forum](https://github.com/geopandas/geopandas/issues/237), but I think it's coming from fiona. Basically people are getting errors installing in an anaconda installation that is only fixed by dumping everything and replacing with full conda-forge installs. 

(Other place this has been discussed before: #1 )

I think it's related to [this gotcha](https://conda-forge.github.io/docs/conda-forge_gotchas.html), but it seems enough people are hitting it I'm wondering if it might be possible to change the listed dependencies for fiona so that it loads the right versions of everything. 

cc: @anastasiaclark @choldgraf @dreyco676

**Comment by ocefpaf** [comment]
> I think that something in the fiona feedstock is causing conflicts with the default anaconda python installation in osx.

Yes. They are not compatible. Unfortunately there is not much we can do besides recommending users to install in a fresh env or to use miniconda. \*

> It's mostly been coming up in the geopandas forum, but I think it's coming from fiona. 

Sorry that this discussion is happening there, but it is a conda limitation IMO because it cannot handle properly the change from all the pre-installed packages in the default Anaconda distribution.

BTW I recommend you to add something like [this](https://github.com/Toblerity/Fiona/blob/master/ISSUE_TEMPLATE.md#fiona-version-and-provenance) to help guide people to conda related issues quicker.

> I think it's related to this gotcha, but it seems enough people are hitting it I'm wondering if it might be possible to change the listed dependencies for fiona so that it loads the right versions of everything.

We should update the `gotcha` page to make the Anaconda distribution issue more clear, but we cannot downgrade our stack of dependencies to satisfy outdated package in `deafults`.

For example: we just upgraded to `hdf5 1.8.18` (used in `gdal` and linked in `fiona`) due to a security reason, while the default channel won't update it for "stability" reasons.

TL;DR

1) use `miniconda` when relying on `conda-forge`'s packages
2) use envs to avoid conflicts
3) request continuum to create `defaults` compatible channel using `conda-forge` recipes

I think that 1 and 2 are more feasible b/c it depends on the end user.

\* We are also working on reducing the gap of this incompatibility, but `conda-forge` evolves at a different pace than `defaults`. See https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L28-L74 for a list of possible issue that can surface due to our pinning difference.
